app/src/main/java/org/fossasia/pslab/adapters/ControlMainAdapter.java

```diff
@@ -64,75 +70,63 @@ public void onBindViewHolder(final ViewHolder holder, int position) {
 
         final Button buttonControlMain1 = holder.buttonControlMain1;
         final Button buttonControlMain2 = holder.buttonControlMain2;
-        Button buttonControlMain3 = holder.buttonControlMain2;
-
-        final SeekBar seekbarControlMain = holder.seekbarControlMain;
-        final EditText edittextControlMain = holder.edittextControlMain;
-
-        buttonControlMain1.setEnabled(false);
-        buttonControlMain2.setEnabled(false);
-
-        edittextControlMain.addTextChangedListener(new TextWatcher() {
```

Comment:
I added this part because when the edittext boxes are empty and the buttons beside them are pressed, there is a crash. Please check it once.